specification/ORCv2.md

```diff
@@ -301,6 +378,9 @@ message ColumnStatistics {
  optional BinaryStatistics binaryStatistics = 8;
  optional TimestampStatistics timestampStatistics = 9;
  optional bool hasNull = 10;
+ optional uint64 bytes_on_disk = 11;
+ optional CollectionStatistics collection_statistics = 12;
```

Comment:
Can we handle these two above lines independently because this is irrelevant to the `geometry`, @wgtmac ?

Reply:
I've created https://github.com/apache/orc-format/pull/22 to address this.